Api/V8/JsonApi/Repository/Filter.php

```diff
@@ -71,24 +78,20 @@ public function parseWhere(\SugarBean $bean, array $params)
             }
         }
 
-        return implode(sprintf(' %s ', $operator), $where);
-    }
-
-    /**
-     * Only return deleted records if they were explicitly requested
-     *
-     * @param array $params
-     * @return array
-     */
-    protected function addDeletedParameter(array $params)
```

Comment:
We wouldn't be able to remove this function in a minor version due to semVer / backwards compatibility. I would suggest to instead mark it as deprecated if it's no longer required.